shepherd.js/tsconfig.json

```diff
@@ -1,9 +1,9 @@
 {
   "$schema": "https://json.schemastore.org/tsconfig",
   "compilerOptions": {
-    "target": "es2021",
-    "module": "esnext",
-    "moduleResolution": "bundler",
+    "target": "es2020",
+    "module": "es2020",
+    "moduleResolution": "node",
```

Comment:
I'm not sure we want to change these settings. Shepherd is not supported to be used in node, but in browser only, so it should be `bundler` still I think.

Reply:
> While few libraries care about supporting a long-past EOL version of Node, maintainers of very popular libraries should be aware that many TypeScript users are still using --moduleResolution node. If they aren’t literally using Node 10, they really should move away from it, but sometimes migrating can be difficult because it introduces new errors caused by incorrect dependency typings—the problem this tool was made to diagnose. Popular libraries often choose to adopt a strategy that allows --moduleResolution node10 to work even without "exports" support.